config/workflows/etdSubmitWF.xml

```diff
@@ -7,32 +7,32 @@
     <label>Student submits completed ETD</label>
   </process>
   <process name="reader-approval" sequence="3" skip-queue="true">
-    <label>Faculty reader has approved (signal from PeopleSoft)</label>
+    <label>Faculty reader(s) have approved (signal from PeopleSoft)</label>
   </process>
   <process name="registrar-approval" sequence="4" skip-queue="true">
     <label>Final Registrar approval (signal from PeopleSoft)</label>
   </process>
   <process name="submit-marc" sequence="5" skip-queue="true">
-    <label>Transfer stub MARC record to Symphony (drop off directory)</label>
+    <label>Create stub MARC record for Symphony; daily crons run on etd app to aggregate these records and make them available to Symphony</label>
     <prereq>registrar-approval</prereq>
   </process>
   <process name="check-marc" sequence="6" skip-queue="true">
-    <label>Daily check to see if MARC record has been created in Symphony</label>
+    <label>Check to see if MARC record has been loaded into Symphony (ILS); if so, updates identityMetadata with ckey and ETD agreement.  Run via cron.</label>
     <prereq>submit-marc</prereq>
   </process>
   <process name="catalog-status" sequence="7" skip-queue="true">
-    <label>Monitor Symphony status until Cataloging is done</label>
+    <label>Check to see if cataloging is done in Symphony (ILS); run via cron</label>
     <prereq>check-marc</prereq>
   </process>
   <process name="other-metadata" sequence="8" skip-queue="true">
-    <label>Generate content, technical and rights metadata</label>
+    <label>Update descMetadata from Symphony, generate content metadata, rights metadata, create a tag.</label>
     <prereq>catalog-status</prereq>
   </process>
   <process name="start-accession" sequence="9" skip-queue="true">
     <label>Transfer control to Common Accessioning</label>
     <prereq>other-metadata</prereq>
   </process>
   <process name="binder-transfer" sequence="10" skip-queue="true" status="skipped">
-    <label>Transfer PDF to binder drop off directory for bound physical copy</label>
+    <label>No longer used.  (Transfer PDF to binder drop off directory for bound physical copy)</label>
```

Comment:
I dare ask: should we drop this step then?

Reply:
no.  It's better to live junk around because of old data -- an object with an uncompleted WF causes problems later.